src/bit_cuda/bit_cuda_transformer.h

```diff
@@ -61,7 +61,7 @@ class BitCudaTransformer : public CfgVisitor<LatticeType>
             
             for (int i = 1; i < node.arguments.size(); i++)
             {
-                last.next_transfer_index = add_transfer_function(node.formal_parameters[0], node.arguments[i]);
+                last.next_transfer_index = add_transfer_function(node.formal_parameters[i], node.arguments[i]);
```

Comment:
shouldnt "i" start as 0 or is it intentional that we start at 1?

Reply:
the 0 index is handled right before the loop